service/src/test/java/bio/terra/externalcreds/services/LinkedAccountServiceTest.java

```diff
@@ -44,6 +44,22 @@ void testGetLinkedAccount() {
         savedLinkedAccount.map(ImmutableLinkedAccount::copyOf).get().withId(Optional.empty()));
   }
 
+  @Test
+  void testGetGA4GHPassport() {
```

Comment:
also test `getGA4GHPassport` when the linked account does not exist and when the linked account does exist but passport does not